kmq.js

```diff
@@ -127,53 +116,50 @@ const help = (message, action) => {
 }
 
 const startGame = (message) => {
-    if (gameInSession) {
+
+    let gameSession = gameSessions[message.guild.id];
+
+    if (gameSession.inSession) {
         message.channel.send("Game already in session.");
         return;
     }
-    gameInSession = true;
-    let query = `SELECT videos.youtube_link as youtubeLink, videos.name, DATE(videos.publish_date) as date, artists.name as artist, videos.video_type as video_type, videos.dead as dead FROM videos INNER JOIN artists on videos.artistID = artists.id WHERE gender = "female" AND video_type = "main" AND dead = "n" ORDER BY views DESC LIMIT 500`;
+
+    let query = `SELECT videos.youtube_link as youtube_link, videos.name, DATE(videos.publish_date) as date, artists.name as artist, videos.video_type as video_type, videos.dead as dead FROM videos INNER JOIN artists on videos.artistID = artists.id WHERE gender = "female" AND video_type = "main" AND dead = "n" ORDER BY views DESC LIMIT 500`;
     db.all(query, (err, rows) => {
         if (err) console.error(err);
         let random = rows[Math.floor(Math.random() * rows.length)];
-        currentSong = random.name;
-        currentArtist = random.artist;
-        currentSongLink = random.youtubeLink;
-        fetchVideoInfo(currentSongLink, (err, videoInfo) => {
-            playSong(currentSongLink, message);
+        gameSession.startRound(random.name, random.artist, random.youtube_link);
+        fetchVideoInfo(gameSession.link, (err, videoInfo) => {
+            playSong(gameSession.link, message);
         })
     })
 }
 
-const sendSongMessage = (message, isQuit) => {
+const sendSongMessage = (message, isForfeit) => {
+    let gameSession = gameSessions[message.guild.id];
     message.channel.send({
         embed: {
             color: RED,
             author: {
-                name: isQuit ? null : message.author.username,
-                icon_url: isQuit ? null : message.author.avatarURL
+                name: isForfeit ? null : message.author.username,
```

Comment:
!end isn't really forfeitng

Reply:
`isQuit`/`isForfeit` is only true when the user calls !stop, not !end. We don't actually call this function when a game is in progress and a user calls !end, I'll fix that